packages/react-components/react-list-preview/stories/List/ListMultipleActions.stories.tsx

```diff
@@ -0,0 +1,187 @@
+  const styles = useStyles();
+
+  return (
+    <ListItem
+      aria-label="iOS App Prototype"
+      value={props.value}
+      className={styles.listItem}
+      checkmark={{ className: styles.checkmark }}
+    >
+      <Card {...props} selected={undefined} tabIndex={-1}>
+        <CardPreview
+          className={styles.grayBackground}
+          logo={<img className={styles.logoBadge} src={resolveAsset('logo3.svg')} alt="Figma app logo" />}
+        >
+          <img className={styles.smallRadius} src={resolveAsset('office1.png')} alt="Presentation Preview" />
+        </CardPreview>
+
+        <CardHeader
+          header={<Text weight="semibold">iOS App Prototype</Text>}
+          description={<Caption1 className={styles.caption}>You created 53m ago</Caption1>}
+          action={
+            <div className={styles.actionsWrapper}>
+              <Button
+                appearance="primary"
+                aria-label="Install"
+                onClick={e => {
+                  e.preventDefault();
+                  alert('Installing!');
+                }}
+              >
+                Install
+              </Button>
+              <Menu>
+                <MenuTrigger disableButtonEnhancement>
+                  <Button
+                    onClick={e => e.preventDefault()}
+                    appearance="transparent"
+                    icon={<MoreHorizontal20Regular />}
+                    aria-label="More actions"
+                  />
+                </MenuTrigger>
+
+                <MenuPopover>
+                  <MenuList>
+                    <MenuItem
+                      onClick={e => {
+                        e.preventDefault();
+                        alert('Clicked menu item');
+                      }}
+                    >
+                      About
+                    </MenuItem>
+                    <MenuItem
+                      onClick={e => {
+                        e.preventDefault();
+                        alert('Clicked menu item');
+                      }}
+                    >
+                      Uninstall
+                    </MenuItem>
+                    <MenuItem
+                      onClick={e => {
+                        e.preventDefault();
+                        alert('Clicked menu item');
+                      }}
+                    >
+                      Block
+                    </MenuItem>
+                  </MenuList>
+                </MenuPopover>
+              </Menu>
+            </div>
+          }
+        />
+      </Card>
+    </ListItem>
+  );
+};
+
+export const ListMultipleActions = (props: Partial<ListProps>) => {
+  const classes = useStyles();
+
+  const [selectedItems, setSelectedItems] = React.useState<Array<string | number>>([]);
+
+  return (
+    <List
+      layout={'grid'}
+      className={classes.list}
+      focusableItems
+      selectable
+      onSelectionChange={(e, data) => setSelectedItems(Array.from(data.selectedItems))}
```

Comment:
```suggestion
      onSelectionChange={(e, data) => setSelectedItems(data.selectedItems)}
```

I am a bit worried if this is required from consumers.

Reply:
its not actually, this is outdated